### PR TITLE
Ria 3306 hearing notification

### PR DESCRIPTION
**Pull request by jkumar-moj**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RIA-3306

### Change description ###
changes to trigger HO notification on listing a case

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/functionalTest/resources/scenarios/RIA-3306-home-office-notifications-for-case-listing.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{
+  "description": "RIA-3306 sends home office notification for list case event",
+  "enabled": "{$featureFlag.isHomeOfficeIntegrationEnabled}",
```
Comment (Lee365) [inline src/functionalTest/resources/scenarios/RIA-3306-home-office-notifications-for-case-listing.json:3]: Need to change to Launch Darkly feature

**Review by Lee365 — CHANGES_REQUESTED** [review]

File: src/functionalTest/resources/scenarios/RIA-3306-home-office-notifications-for-case-listing.json (hunk 1 above)
Comment (Lee365) [reply src/functionalTest/resources/scenarios/RIA-3306-home-office-notifications-for-case-listing.json]: Added LD feature

**Review by Lee365 — APPROVED** [review]
LGTM

**Review by Lee365 — DISMISSED** [review]
LGTM

**Review by Srihari-Nakkala — APPROVED** [review]
LGTM

**Review by Lee365 — APPROVED** [review]
LGTM

**Pull request merged**